crates/apollo-compiler/test_data/diagnostics/0050_subscription_operation_with_root_introspection_field.graphql

```diff
@@ -0,0 +1,3 @@
+subscription sub {
+  __typename
+}
```

Comment:
Let's add a subscription type to this document so we don't also get a `UnsupportedOperation` diagnostic in our output

```suggestion
subscription sub {
  __typename
}

type Subscription {
  __typename: String
}
```